CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
 ## Master
```

Comment:
`CHANGELOG` looks good this time around, but how about bringing back that [migration guide](https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/28/files/7456b35a17b6e0f220243e619371088eeda72be2#diff-8d9339710527aa49bfd592700562a0f1)?

Reply:
Forgot to bring it in. Migration guide is now back.